bom/pom.xml

```diff
@@ -248,6 +249,12 @@
                 <version>${project.version}</version>
                 <scope>provided</scope>
             </dependency>
+            <dependency>
+                <groupId>io.quarkus</groupId>
+                <artifactId>quarkus-flyway</artifactId>
+                <version>${project.version}</version>
+                <scope>provided</scope>
```

Comment:
the "runtime" flavour of extensions should no longer be marked as provided. Sorry about that, it just changed

Reply:
Done